docs/performance_model.md

```diff
@@ -10,80 +10,89 @@
+| otlp | batch | datadog | datadog_exporter_trace_mock | otlp | m5.2xlarge | 4.94 | 70.06 |
+| otlp | batch | logzio | logzio_exporter_trace_mock | otlp | m5.2xlarge | 3.60 | 83.51 |
+| otlp | batch | newrelic | newrelic_exporter_trace_mock | otlp | m5.2xlarge | 3.75 | 69.99 |
+| otlp | batch | otlp | otlp_grpc_exporter_trace_mock | otlp | m5.2xlarge | 2.88 | 124.58 |
+| otlp | batch | otlphttp | otlp_http_exporter_trace_mock | otlp | m5.2xlarge | 3.04 | 65.35 |
+| otlp | batch | awsxray | otlp_mock | otlp | m5.2xlarge | 3.69 | 64.48 |
+| otlp | batch | awsxray | otlp_trace | otlp | m5.2xlarge | 3.80 | 66.67 |
+| otlp | batch | sapm | sapm_exporter_trace_mock | otlp | m5.2xlarge | 3.79 | 76.93 |
+| zipkin | batch | logging, otlphttp | zipkin_mock | zipkin | m5.2xlarge | 0.04 | 53.79 |
 
 ### Trace (TPS: 1000)
 | Receivers | Processors | Exporters | Test Case | Data Type | Instance Type | Avg CPU Usage (Percent) | Avg Memory Usage (Megabytes) |
 |:---------:|:----------:|:---------:|:---------:|:---------:|:-------------:|:-----------------------:|:----------------------------:|
-| awsxray | batch | awsxray | xrayreceiver | otlp | m5.2xlarge | 0.05 | 51.40 |
-| otlp | batch | datadog | datadog_exporter_trace_mock | otlp | m5.2xlarge | 26.12 | 71.88 |
-| otlp | batch | logzio | logzio_exporter_trace_mock | otlp | m5.2xlarge | 29.54 | 119.02 |
-| otlp | batch | newrelic | newrelic_exporter_trace_mock | otlp | m5.2xlarge | 28.32 | 67.41 |
-| otlp | batch | otlp | otlp_grpc_exporter_trace_mock | otlp | m5.2xlarge | 20.58 | 1074.97 |
-| otlp | batch | otlphttp | otlp_http_exporter_trace_mock | otlp | m5.2xlarge | 20.46 | 63.43 |
-| otlp | batch | awsxray | otlp_mock | otlp | m5.2xlarge | 35.48 | 69.38 |
-| otlp | batch | awsxray | otlp_trace | otlp | m5.2xlarge | 32.66 | 71.27 |
-| otlp | batch | sapm | sapm_exporter_trace_mock | otlp | m5.2xlarge | 22.27 | 78.60 |
-| otlp | batch | splunk_hec | splunkhec_exporter_trace_mock | otlp | m5.2xlarge | 25.60 | 66.98 |
+| awsxray | batch | awsxray | xrayreceiver | xray | m5.2xlarge | 26.98 | 74.04 |
+| jaeger | batch | otlphttp | jaeger_mock | jaeger | m5.2xlarge | 0.04 | 54.56 |
+| otlp | batch | datadog | datadog_exporter_trace_mock | otlp | m5.2xlarge | 29.16 | 68.68 |
+| otlp | batch | logzio | logzio_exporter_trace_mock | otlp | m5.2xlarge | 25.88 | 117.45 |
+| otlp | batch | newrelic | newrelic_exporter_trace_mock | otlp | m5.2xlarge | 26.92 | 67.16 |
+| otlp | batch | otlp | otlp_grpc_exporter_trace_mock | otlp | m5.2xlarge | 24.79 | 663.47 |
+| otlp | batch | otlphttp | otlp_http_exporter_trace_mock | otlp | m5.2xlarge | 22.86 | 65.82 |
+| otlp | batch | awsxray | otlp_mock | otlp | m5.2xlarge | 27.31 | 67.23 |
+| otlp | batch | awsxray | otlp_trace | otlp | m5.2xlarge | 28.76 | 69.97 |
+| otlp | batch | sapm | sapm_exporter_trace_mock | otlp | m5.2xlarge | 23.01 | 80.44 |
+| zipkin | batch | logging, otlphttp | zipkin_mock | zipkin | m5.2xlarge | 0.04 | 55.50 |
```

Comment:
```suggestion
| zipkin | batch | logging, otlphttp | zipkin_mock | zipkin | m5.2xlarge | 23.56 | 226.11 |
```

Reply:
Is this PR generated by bot? Then why we want to modify the per values? :dog: